dir.props

```diff
@@ -26,6 +26,11 @@
       <Value>True</Value>
     </AssemblyMetadata>
   </ItemGroup>
+  
+  <!-- Temporary adding this property so that Linux builds will target the net45 version of the task library instead of the .Net Core version -->
```

Comment:
Got it, this is talking about building on non-Windows rather than for non-Windows.  Thanks.